spec/dummy/db/schema.rb

```diff
@@ -309,6 +322,11 @@
   add_foreign_key "atmosphere_http_mappings", "atmosphere_compute_sites", name: "atmosphere_http_mappings_compute_site_id_fk", column: "compute_site_id"
   add_foreign_key "atmosphere_http_mappings", "atmosphere_port_mapping_templates", name: "atmosphere_http_mappings_port_mapping_template_id_fk", column: "port_mapping_template_id"
 
+  add_foreign_key "atmosphere_migration_jobs", "atmosphere_appliance_types", name: "atmo_mj_at_fk", column: "appliance_type_id"
+  add_foreign_key "atmosphere_migration_jobs", "atmosphere_compute_sites", name: "atmo_mj_csd_fk", column: "compute_site_destination_id"
```

Comment:
Line is too long. [136/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.